### ISSUE TITLE
The Last updated date doesn't work in production enviroment

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#336]
I will fill out the exact info later, this is just a reminder for me not to forget 

- [ ] I've checked the [existing issues for this change](https://github.com/wpaccessibility/wp-a11y-docs/issues)

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by joedolson** [comment]
Based on my research, the problem is with the fetch depth - Github by default executes Jekyll with fetch-depth of 1, which means that the file history is not available for `last_updated_at` to render. It works locally because the local set up has the full Git history available. 

I don't know any way to test this other than to commit it, however.

**Comment by joedolson** [comment]
Looks to me like this is now working with #359; so closing this out!
